### DISCUSSION TITLE
custom theme with auto switch theme

### DISCUSSION THREAD

**Discussion by ninofaivre** [monkeytypegame/monkeytype#5218] [Ideas]
Hi, I'm using [catppuccin latte](https://monkeytype.com/?customTheme=WyIjZWZmMWY1IiwiIzQwYTAyYiIsIiNkYzhhNzgiLCIjYWNiMGJlIiwiI2U2ZTllZiIsIiM0YzRmNjkiLCIjZDIwZjM5IiwiI2U2NDU1MyIsIiNkMjBmMzkiLCIjZTY0NTUzIl0=) and [catppuccin machiato](https://monkeytype.com/?customTheme=WyIjMjQyNzNhIiwiI2E2ZGE5NSIsIiNmNGRiZDYiLCIjNWI2MDc4IiwiIzFlMjAzMCIsIiNjYWQzZjUiLCIjZWQ4Nzk2IiwiI2VlOTlhMCIsIiNlZDg3OTYiLCIjZWU5OWEwIl0=) which are not included in monkeytype themes. I like to have all my apps switching between light and dark based on the current hour. There is already the *auto switch theme* option but it doesn't works with custom themes.

**Comment by sanidhyas3s** [comment]
Fair concern. I think we can add the custom options to the auto-switch settings as well. Only downside I can think is it will make the auto-switch settings look a bit more complicated.

Also, we should add catppuccin themes to the preset themes anyway for their popularity.
Source: https://github.com/catppuccin/monkeytype

**Reply by SameerJS6** [reply]
https://github.com/monkeytypegame/monkeytype/discussions/7041 might wanna take a look at this? And if possible, give some feedback?

**Comment by stellophiliac** [comment]
kind of old discussion, but i'd really like this

**Reply by SameerJS6** [reply]
https://github.com/monkeytypegame/monkeytype/discussions/7041 can you please see this once? And if possible, give some feedback?
